public/js/main.js

```diff
@@ -1,69 +1 @@
-const deleteBtn = document.querySelectorAll('.del')
```

Comment:
holup i'm really tired, we still need all of this don't we?

Reply:
All of the previous content in Main.js was just todo list stuff copied in from Leons code, we haven't used it.